py/setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = serialkeel
+version = 0.1.0
```

Comment:
I think you can leave this out. As long as you tag a commit and use `python3 -m build`, that will grab the latest commit/tag and use that for versioning :)

Reply:
If some commit is tagged, and we `python -m build` on the next commit, what is the resulting version?

Reply:
Not sure if I'm missing something simple, but I was just trying to get this automatic versioning to work locally with a git tag, but it just filled in with 0.0.0. I suspect that there might be more to do to make this work.

I read this, so it may be possible with a little more work: https://stackoverflow.com/questions/5581722/how-can-i-rewrite-python-version-with-git
If there is no tag on the current commit, we can allow it to fall back to a default.

I think we can stick to just specifying the version if it can't automatically pick up the version from the tag?

Reply:
Disregard the previous comment, I just figured this out. Needed to add a few more lines to the pyproject.toml file. Thanks @Mangl10!

@torsteingrindvik If a commit is tagged as v0.1.0, a commit or more later, it automatically tags it as 0.1.0.post0, post1 etc. I think this works?

Reply:
Ok, that sounds good. Just making sure it's not easy to make collisions.